E3Next/Settings/CharacterSettings.cs

```diff
@@ -337,7 +342,17 @@ private void LoadData()
 
 
             LoadKeyData("Heals", "Tank", ParsedData, HealTankTargets);
+            for (Int32 i = 0; i < HealTankTargets.Count; i++)
+            {
+                HealTankTargets[i] = e3util.FirstCharToUpper(HealTankTargets[i]);
```

Comment:
Why not just use `TextInfo.ToTitleCase()`?

Reply:
https://itecnote.com/tecnote/c-make-first-letter-of-a-string-upper-case-with-maximum-performance/